src/style.css

```diff
@@ -143,6 +144,7 @@ body {
     position: relative;
     border-radius: 0.5rem 0.5rem;
     border-top-left-radius: 0;
+    z-index: 200;
```

Comment:
can you explain why `z-index` must be used?

Reply:
@zamhaq , initially i was trying to append via DOM ,but i'll change it ASAP.